CONTRIBUTING.md

```diff
@@ -2,23 +2,26 @@
 
 ## Project setup
 
-1. If you don't have `Poetry` installed run:
+1. If you don't have `uv` installed run:
 
 ```bash
-make poetry-install
+make setup
 ```
 
-> This installs Poetry as a [standalone application][poetry-install]. <br>
-> For more details, see also the [Poetry Documentation][poetry-docs]. <br>
-> If you prefer, you can simply install it inside your virtual environment.
+> This installs uv as a [standalone application][uv-install]. <br>
+> For more details, see also the [uv documentation][uv-docs]. <br>
 
-2. Initialize project dependencies with poetry and install `pre-commit` hooks:
+2. Initialize project dependencies with uv and install `pre-commit` hooks:
 
 ```bash
 make install-dev
 make pre-commit-install
 ```
 
+This will install project dependencies into the currently active environment. If you would like to 
+use uv's default behavior of managing a project-scoped environment, use uv commands directly to 
```

Comment:
use the backticks for `uv` everything it is mentioned for consistency

Reply:
No, it should be the other way around. Markdown should highlight that this is a command rather than any other word.

Reply:
I think it is more conventional to refer to the tool like the providers of the tool do in their docs. Poetry, uv, etc. don't use backticks. likewise, I don't use backticks when documenting or discussing Xarray or any other python library unless I am using it directly in a code sample.

Reply:
Don't mind that much but think it makes more sense to keep it as it is now. we didn't use backticks for poetry unless it was in a code sample.